Makefile

```diff
@@ -0,0 +1,101 @@
+TOOLS_DIR="$(PWD)/.tools"
```

Comment:
should we use make on CI?

Reply:
It can be done, but common steps in github workflows, e.g. linter step, are usually having their own configuration and way to run them.